### DISCUSSION TITLE
Updating the release plan

### DISCUSSION THREAD

**Discussion by benjaoming** [django-wiki/django-wiki#1286] [General]
We currently have a release plan that doesn't include the current and next series :)

 * 0.3.x series supports Django 1.11. As with the upstream Django release, 0.3 was be the last series with Python 2.7 support.
 * 0.4.x supports Django 1.11 and Django 2.1 and Python 3.4+.
 * 0.5.x Remove Django 1.11 support, adds Django 2.2 and 3.x support. Python 3.5+.
 * 0.6.x Targets Bootstrap v4, if you are interested in this work, please get in touch on Github!
 * 0.7.x Removes Django 2.1 support, adds Django 3.1, 3.2
 * 0.9.x Definitely Removes Python 3.5 and 3.6 support

I could suggest the following to be added?

 * 0.10.x series uses Hatch as the new build system and has upgrades for Django 4.x and Python 3.11.
 * 0.11.x will upgrade Bootstrap to the 5.x series. Removes Python 3.7 support.

What do you think @oscarmcm ? I know that you mentioned Bootstrap earlier? I can sort of commit to that as well, depends on what you're interested in working on :)

**Comment by oscarmcm** [comment]
@benjaoming this is what I think for 0.11.x
 
 - #1224 
 - #1118 
 - Finish #1274 
 - Update to BS 5x
 - Improve static files (probably drop sass dependency)
 - Drop py3.7 support

**Reply by benjaoming** [reply]
I've created a 0.11 milestone :+1: 

Once 0.10 is released, we can say that `main` targets 0.11 and split out a `releases/0.10.x` branch (that pattern has been used earlier) - doesn't mean we'll necessarily need to do more 0.10 releases, but it's always good to have the option.
